### PR TITLE
#68 fix navbar

### LINKED ISSUE #68: Make Nav Drawer logo and heading size proportional to Navbar
The nav drawer logo and other headings are slightly bigger than the main navbar content

### PR DESCRIPTION
**Pull request by SharmaTushar1**
Closes #68 

Had to change the .eslintrc.json and .prettierrc files. See [this](https://stackoverflow.com/questions/53516594/why-do-i-keep-getting-eslint-delete-cr-prettier-prettier?page=1&tab=scoredesc#tab-top) and [this](https://stackoverflow.com/questions/34215526/eslint-how-to-disable-unexpected-console-statement-in-node-js).

### CODE REVIEW

**Comment by nitishdhar** [comment]
Can you attach screenshots of the fix. Thanks

File: .eslintrc.json [hunk 1]
```diff
@@ -7,7 +7,13 @@
...
   "rules": {
     "no-unused-vars": [1, { "args": "after-used", "argsIgnorePattern": "^_" }],
-    "prettier/prettier": "error",
+    "prettier/prettier": [
+      "error",
+      {
+        "endOfLine": "auto"
+      }
+    ],
+    "no-console": "off",
```
Comment (nitishdhar) [inline .eslintrc.json]: We don’t want to turn off this rule. Instead remove console statements over time as the app matures and we don’t need to commit console log statements.

File: app/_components/nav-drawer/NavDrawer.tsx [hunk 2]
```diff
@@ -80,7 +80,7 @@ const NavDrawer = () => {
             backgroundColor={selectColor(isDark, 'box')}
             color={selectColor(isDark, 'textLight')}
           >
-            <Heading as="h1" size="2xl">
+            <Heading as="h1" size="xl">
```
Comment (nitishdhar) [inline app/_components/nav-drawer/NavDrawer.tsx]: Remove Menu, it’s redundant

**Comment by SharmaTushar1** [comment]
> Can you attach screenshots of the fix. Thanks

image

File: .eslintrc.json (hunk 1 above)
Comment (SharmaTushar1) [reply .eslintrc.json]: I have pushed new changes. I have removed the `"no-console:"off"` rule. Should I also remove the prettier `"endOfLine":"auto"`? Earlier it was throwing [warnings and errors](https://stackoverflow.com/questions/53516594/why-do-i-keep-getting-eslint-delete-cr-prettier-prettier?page=1&tab=scoredesc#tab-top) without it.

**Pull request closed without merging**